dev-java/jmock/jmock-1.2.0-r2.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Library for testing Java code using mock objects"
+HOMEPAGE="http://jmock.org/"
+SRC_URI="http://jmock.org/downloads/${P}-jars.zip"
+
+LICENSE="BSD"
+SLOT="1.0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos"
+
+DEPEND="dev-java/junit:0"
+RDEPEND="${DEPEND}
+	>=virtual/jre-1.8
+	"
+BDEPEND="${DEPEND}
+	app-arch/unzip
+	>=virtual/jdk-1.8
```

Comment:
we don't put jdk in BDEPEND but in DEPEND. from the logic the package uses classes from the jdk and "links" against them so it belongs to DEPEND instead. also, for both jre/jdk it should specify `:*` at the end of the dep, so `>=virtual/jdk-1.8:*` and `>=virtual/jre-1.8:*`.